.travis.yml

```diff
@@ -1,3 +1,6 @@
-jdk: openjdk6
+jdk:
+  oraclejdk8
+  openjdk7
+  openjdk6
```

Comment:
Btw, written like this, this doesn't actually test openjdk7 or openjdk6 any more, as apparently it sticks it all on one line and then casually throws anyway any unnecessary, redundant arguments:

```
$ jdk_switcher use oraclejdk8 openjdk7 openjdk6
Switching to Oracle JDK8 (java-8-oracle), JAVA_HOME will be set to /usr/lib/jvm/java-8-oracle
```

You want to write this instead:
```yml
jdk:
  - oraclejdk8
  - openjdk7
  - openjdk6
```